tests/router/test_router_e2e_with_mockers.py

```
@@ -947,6 +973,279 @@ def test_router_decisions_disagg(
+        shared_namespace,
+        registration_order="prefill_first",
+        prefill_mocker_args=prefill_mocker_args,
+        decode_mocker_args=decode_mocker_args,
+        num_prefill_mockers=1,
+        num_decode_mockers=1,
+        enable_disagg_bootstrap=True,
+        request_plane=request_plane,
+        event_plane="nats",
+        prefill_env_overrides=prefill_env,
+        decode_env_overrides=decode_env,
+    ) as (prefill_workers, decode_workers):
```

Ruff (0.15.20)

[warning] Unpacked variable `prefill_workers` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)